embedded-visualization/Virtual Machines/cspe-VM-Capacity-and-Cost.json

```diff
@@ -419,12 +419,11 @@
         "allValue": null,
         "current": {
           "selected": true,
-          "tags": [],
           "text": [
-            "All"
+            "christophe.sperandio"
```

Comment:
same question

Reply:
Yes, correct, should be "All", that's better for initial sharing.